### PR TITLE
Adding mailer

### PR DESCRIPTION
**Pull request by bloodbare**

### CODE REVIEW

File: guillotina/contrib/mailer/__init__.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from guillotina.interfaces import IMailer
+from guillotina.component import provide_utility
+from guillotina.utils import import_class
+import logging
+
+logger = logging.getLogger('guillotina.contrib.mailer')
+
+app_settings = {
+    "mailer": {
+        "default_sender": "foo@bar.com",
```
Comment (vangheem) [inline guillotina/contrib/mailer/__init__.py]: Should we leave `deault_sender` to None and log warning if not provided and trying to send?

File: guillotina/contrib/mailer/__init__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "endpoints": {
+            "default": {
+                "type": "smtp",
```
Comment (vangheem) [inline guillotina/contrib/mailer/__init__.py]: Same here I wonder?

File: guillotina/contrib/mailer/__init__.py (hunk 1 above)
Comment (bloodbare) [reply guillotina/contrib/mailer/__init__.py]: agree

File: guillotina/contrib/mailer/__init__.py (hunk 2 above)
Comment (bloodbare) [reply guillotina/contrib/mailer/__init__.py]: I would keep localhost as default option

**Comment by bloodbare** [comment]
Yups, we should have it, can we plan to add on beta phase ?

**Pull request merged**